app/javascript/app/pw-toggle.js

```diff
@@ -10,11 +10,11 @@ function togglePw() {
       const el = `
         <div class="top-n24 right-0 absolute">
           <label class="btn-border" for="pw-toggle-${i}">
-            <div class="checkbox">
+            <span class="checkbox">
```

Comment:
For what it's worth, these are being replaced altogether as part of #5279. But I can address conflicts in my branch if this is merged first.